ptf/sai_base_test.py

```diff
@@ -34,7 +34,7 @@
 from sai_thrift import sai_rpc
 
 from sai_utils import *
-import sai_adapter as adapter
+import sai_thrift.sai_adapter as adapter
```

Comment:
@aljer @pkochmanska 
In this new sai-ptf structure, looks like there are two kinds of files, handwriting code, and auto-generated code.
in order to make the test cases related files to import the generated files from an installed package, try to organize those generated files into a package.
Please help make a review, not sure if this will impact the design of this sai-ptf structure.

thanks.

Reply:
You mean this has to be incorporated into all test files?

Reply:
Yes, this will impact all the test files.
Cause we import the sai_adapter in the sai_base_test, we don't need to change every test case.
the purpose of this change is to install all the dependent files into an installation package,(certainly, we can install those files under dist-package directly), and give it a naming space.

Reply:
I think we can update the name space. It will be more explicit